frontendservice/src/views/CustomerHome.vue

```diff
@@ -7,48 +7,168 @@ CONDITIONS OF ANY KIND, either express or implied. See the License for the
 specific language governing permissions and limitations under the License. */
 
 <template>
-  <div class="home">
-    <h1>Welcome customer!</h1>
-    <p>
-      This page will be Home page for the customers. 
-    </p>
-    <p>
-      ToDo: Implement CustomerHome page features including NavBar, Search Bar, MyTokens and AllShops.
-    </p>  
-    <div>
-      <router-link to="/my-tokens">My Tokens</router-link>
+  <div v-if="!isLoggedIn">
+    <NavBarCustomer></NavBarCustomer>
+    <cookieinfo></cookieinfo>
+    <cookieackno></cookieackno>
+    <div id="searchBarWrap" style="margin: 20px 0">
+      <div class="field has-addons">
+        <div class="form-group has-feedback">
+          <i class="fa fa-search" style="font-size:32px"></i>
+          <input id="searchBar" class="input" type="text" placeholder="Find a shop" />
+        </div>
+      </div>
     </div>
-    <div v-if="!isLoggedIn">
-      <router-link to="/about">Back</router-link>
+    <div id="wrap" class="control">
+      <div class="select">
```

Comment:
Is this acting as a filter for the search bar? If so, did we not plan to remove it? Correct me if I'm wrong, I'm might be not fully aware. Ishita might be having a better idea on this.

Reply:
Yes rightly pointed. Was planning to change this after Ishita's shop owner home pr get's merged.

Reply:
What did we conclude here?

Reply:
We co-interns are going to have a call later to decide on this.

Reply:
Ishita's PR has been merged I guess, so you can refer to her search bar and stuff related to that in her file.

Reply:
Ishita's PR has been merged I guess, so you can refer to her search bar and stuff related to that in her file.

Reply:
Are we waiting for some discussion here?

Reply:
Added TODO, will change it later